<launch>

  <rosparam file="$(find asv_ros_revamp)/config/params.yaml" command="load" />
    
  <node name="data_acq" pkg="asv_ros_revamp" type="data_acq" output="screen"/>
  <node name="navigation_node" pkg="asv_ros_revamp" type="navigation_node" output="screen"/>
  <node name="safety_node" pkg="asv_ros_revamp" type="safety_node"/>
  <node name="control_node" pkg="asv_ros_revamp" type="control_node" output="screen"/>
  <node name="thruster_node" pkg="asv_ros_revamp" type="thruster_node" output="screen"/>


</launch>
<!-- respawn="true"  to node for auto restart on node crash -->